Makefile.common

```diff
@@ -463,7 +463,7 @@ endif
 
 # Soundfonts
 DIST_FILES_SOUNDFONTS=
-ifdef USE_FLUIDSYNTH
+ifneq "$(or $(USE_FLUIDSYNTH),$(USE_FLUIDLITE))" ""
```

Comment:
This should go in its own commit as it's touching more than Emscripten

Reply:
I split this into a separate commit